node/parallel/src/chain_spec/heiko.rs

```diff
@@ -74,8 +76,8 @@ pub fn development_config(id: ParaId) -> ChainSpec {
         },
         vec![],
         None,
-        None,
-        None,
+        Some("dev"),
```

Comment:
Heiko is not a dev chain.

Reply:
Ok, I know, but it's in development config not in live conifg. Do we share the config in both network?

Reply:
This is protocol-id right? if it's public network, should be `heiko`, public testnet `heiko-staging`, local network should be `heiko-dev` and `heiko-local`.